root/etc/e-smith/templates/etc/shorewall/shorewall.conf/60options

```diff
@@ -20,12 +20,10 @@ AUTOCOMMENT=Yes
 
 AUTOHELPERS=Yes
 
-AUTOMAKE=No
+AUTOMAKE=Yes
```

Comment:
@filippocarletti @gsanchietti after substituting my NICs shorewall configuration keeps references to old interfaces and fails to start

```
[root@vm5 ~]# shorewall start
Starting Shorewall....
Initializing...
Processing /etc/shorewall/init ...
Processing /etc/shorewall/tcclear ...
Setting up Route Filtering...
Setting up Martian Logging...
Setting up Proxy ARP...
   WARNING: Optional Interface eth3 is not usable -- eth3 not Started
Setting up Traffic Control...
   WARNING: Device eth3 is not in the UP state -- traffic-shaping configuration skipped
Preparing iptables-restore input...
Running /sbin/iptables-restore --wait 60...
```

Old configuration is retained under `/var/lib/shorewall/`